src/vsg/core/Data.cpp

```diff
@@ -51,7 +51,7 @@ Data::MipmapOffsets Data::computeMipmapOffsets() const
     offsets.push_back(lastPosition);
     while (numMipmaps > 1 && (w > 1 || h > 1 || d > 1))
     {
-        lastPosition += (w * h * d);
+        lastPosition += (static_cast<std::size_t>(w) * h * d);
```

Comment:
I think it would be better to replace the auto w = width() etc. with std::size_t rather than have a cast on an inner loop.